src/components/RightPane/SectionTable/SectionTableButtons.tsx

```diff
@@ -91,6 +91,25 @@ export const ScheduleAddCell = withStyles(styles)((props: ScheduleAddCellProps)
         }
     };
 
+    const closeCopyAndAlert = () => {
+        const url = new URL(window.location.href)
+        const urlParam = new URLSearchParams(url.search);
+        urlParam.delete('courseCode');
+        urlParam.delete('courseNumber');
+        urlParam.delete('deptLabel');
+        urlParam.delete('deptValue');
+        urlParam.delete('GE');
+        urlParam.delete('term');
```

Comment:
Do these have any significance? i.e. are they all of the form values or just some specific ones? Here's a revised approach if it's only subset of the form fields.
```suggestion
       // somewhere at the top of the file or something, 
       // and include a description, i.e. why they need to be reset
       const fieldsToReset = ['courseCode', 'courseNumber', 'deptLabel', 'deptValue', 'GE', 'term']
       fieldsToReset.forEach(field => urlParam.delete(field))
```